### ISSUE TITLE
Remove Django 1.11 cache_get_or_set backport

### ISSUE THREAD

**Issue by EnTeQuAk** [mozilla/addons-server#13236]
We backported the `cache.get_or_set` function from Django 1.11 as `olympia.lib.cache:cache_get_or_set`

We're now way past that, let's use the official version.

https://docs.djangoproject.com/en/3.0/topics/cache/#django.core.caches.cache.get_or_set

**Comment by dbxnr** [comment]
Can I have a go at this?

**Comment by Shivansh2407** [comment]
Can You Please guide me sir how to proceed with this issue sir @EnTeQuAk ?

**Comment by ioanarusiczki** [comment]
@EnTeQuAk Is this a qa not neeeded? In case qa could do some testing on it, please add some str
Thanks!

**Comment by EnTeQuAk** [comment]
There isn't much to specifically test for this. The code that has been changed primarily covers the reviewers api which has been fixed by https://github.com/mozilla/addons-server/issues/13260 as well as the old file-viewer and user blocking via `DeniedName`

Your regular tests will probably cover this.

**Comment by ioanarusiczki** [comment]
@EnTeQuAk 
- the reviewers API have been tested for https://github.com/mozilla/addons-server/issues/13260 and no issues were found.
- I did not test much around the old file viewer since the new code manager is in scope now. I've noticed issues here:
- the keyboard nav is not working when I try to view the next or previous file. 
- when I'm in a compare page and I click a file from the file tree the browse page will load instead. 

Let me know if issues should be filed - I will investigate then in more detail everything. 

- the denied name from admin tools works with the denied names which were previously added. 
To give an example: on dev there were 3 names. I tested with all of them trying to add those as the Display Name for a new account but it won't let me finish editing the user because I receive a red warning message " This display name cannot be used."
I added today two more: `statistics` and `edit` and tried with these too but this time it let me use them as display names. Same on stage, for `bimbambum` the one I added today.

If I should file an issue let me know.
